src/docs/team-setup.md

```diff
@@ -115,13 +115,13 @@ Next, navigate to [Account details](https://ci.appveyor.com/account) page and ch
 On the same page authorize AppVeyor to access GitHub. A new OAuth token will be generated and used for accessing organization's repositories and teams on bahalf of your GitHub user.
 Make sure your GitHub user has owner/admin rights on the repositories you are going to add later as AppVeyor projects.
 
-**Add yourself as co-administrator**
+#### Add yourself as co-administrator
 
 Go to [Team](https://ci.appveyor.com/team) page and add a new collaborator with primary email address of your GitHub account and *Administrator* role.
 **Sign out from AppVeyor**. Login again, but now with "GitHub" button - you will be presented with dropdown displaying two accounts - your personal
 account and {% raw %}<code>{organization}</code>{% endraw %} account you just created. Select {% raw %}<code>{organization}</code>{% endraw %} account and click "GitHub" sign in button again.
 Now you are logged in with your GitHub user and are managing "organizational" account.
 
-**Add your co-workers**
+#### Add your co-workers
 
```

Comment:
We should define a style for `<h4>` then; otherwise it looks similar to h3:

![image](https://cloud.githubusercontent.com/assets/5041459/17940485/42e27c46-69e4-11e6-9ebd-1650c7426b67.png)